lib/src/platform_file.dart

```diff
@@ -31,21 +31,9 @@ class PlatformFile {
   /// ```
   /// final File myFile = File(platformFile.path);
   /// ```
-  /// On web this is always `null`. You should access `bytes` property instead.
+  /// On web the path points to a Blob URL, if present.
```

Comment:
Should we perhaps tell users to call `URL.revokeObjectURL` on the path when they are done with the file (on web)?

```suggestion
/// On web the path points to a Blob URL, if present, which can be cleaned up using [URL.revokeObjectURL].
```